blazor/file-manager/context-menu.md

```diff
@@ -0,0 +1,322 @@
+                        args.Items[j].Disabled = false;
+                    }
+                }
+            }
+        }
+        
+    }
+}
+
+```
+
+
+## Showing or hiding items
+
+In the Blazor FileManager component, you can control the visibility of context menu items by setting the [Hidden](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.MenuItemModel.html#Syncfusion_Blazor_FileManager_MenuItemModel_Hidden) value of the [MenuOpened](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerEvents-1.html#Syncfusion_Blazor_FileManager_FileManagerEvents_1_MenuOpened) event arguments to **true** or **false**. 
+
+In the following example, the **Cut** context menu item is shown for the files.
+
+```cshtml
+
+@using Syncfusion.Blazor.FileManager
+
+<SfFileManager TValue="FileManagerDirectoryContent" >
+    <FileManagerEvents TValue="FileManagerDirectoryContent"></FileManagerEvents>
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/GetImage">
+    </FileManagerAjaxSettings>
+    <FileManagerEvents TValue="FileManagerDirectoryContent" MenuOpened="MenuOpened"></FileManagerEvents>
+</SfFileManager>
+
+@code {
+
+    public void MenuOpened(MenuOpenEventArgs<FileManagerDirectoryContent> args)
+    {
+        for (int i = 0; i < args.FileDetails.Count(); i++)
+        {
+            if (!args.FileDetails[i].IsFile)
+            {
+                for (int j = 0; j < args.Items.Count(); j++)
+                {
+                    if (args.Items[j].Text == "Cut")
+                    {
+                        args.Items[j].Hidden = true;
+                    }
+                }
+            }
+            else
+            {
+                for (int j = 0; j < args.Items.Count(); j++)
+                {
+                    if (args.Items[j].Hidden == true)
+                    {
+                        args.Items[j].Hidden = false;
+                    }
+                }
+            }
+        }
+        
+    }
+}
+
+```
+
+## Events
+
+The Blazor FileManager Context Menu component has a [MenuOpened](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerEvents-1.html#Syncfusion_Blazor_FileManager_FileManagerEvents_1_MenuOpened) and [OnMenuClick](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerEvents-1.html#Syncfusion_Blazor_FileManager_FileManagerEvents_1_OnMenuClick) events that can be triggered for certain actions. These events can be attached to the FileManager using the **FileManagerEvents** component, which requires the **TValue** to be provided.
+

```

Comment:
These events can be bound to the FileManager using the **FileManagerEvents**, which requires the **TValue** to be provided.

Reply:
Done.